### ISSUE TITLE
hwdb: Error calling EVIOCSKEYCODE on "Dell WMI hotkeys"

### ISSUE THREAD

**Issue by andreyv** [systemd/systemd#4980]
### Submission type

  - [x] Bug report
  - [ ] Request for enhancement (RFE)

### systemd version the issue has been seen with

232

### Used distribution

Arch Linux

### In case of bug report: Unexpected behaviour you saw

On a Dell Latitude E5550, firmware version A13 and Linux version 4.9.0, udevd prints the following error to the journal on boot:

    systemd-udevd[311]: Error calling EVIOCSKEYCODE on device node '/dev/input/event17' (scan code 0x150, key code 190): Invalid argument

`/dev/input/event17` is
```
I: Bus=0019 Vendor=0000 Product=0000 Version=0000
N: Name="Dell WMI hotkeys"
P: Phys=wmi/input0
S: Sysfs=/devices/virtual/input/input18
U: Uniq=
H: Handlers=kbd event17 
B: PROP=0
B: EV=13
B: KEY=800000000000 0 0 101100b00000c00 200300000 e000000000000 0
B: MSC=10
```

The same bug is reported at https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1597415. According to a comment there, these lines in `hwdb/60-keyboard.hwdb` seem to be related:
```
# Dell Latitude microphone mute
evdev:name:Dell WMI hotkeys:dmi:bvn*:bvr*:bd*:svnDell*:pnLatitude*
# Dell Precision microphone mute
evdev:name:Dell WMI hotkeys:dmi:bvn*:bvr*:bd*:svnDell*:pnPrecision*
KEYBOARD_KEY_150=f20 # Mic mute toggle, should be micmute
```

**Comment by poettering** [comment]
@andreyv hmm, i wonder if your system actually has the key in question, i.e. a mic button?

@martinpitt any idea?

**Comment by andreyv** [comment]
It does have a "Disable microphone" key (Fn + F4), which, in fact, doesn't work. I also don't see any events from this key with `xev`.

**Comment by andreyv** [comment]
Fixed in #5012.
